### ISSUE TITLE
npm start causes cmd window to crash 

### ISSUE THREAD

**Issue by EnterAUserName12** [ionic-team/stencil#2999]
**Stencil version:**
-- @stencil/core@2.6.0

**I'm submitting a:**

[x ] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://stencil-worldwide.herokuapp.com/ or https://forum.ionicframework.com/

**Current behavior:**
running npm start causes the component to be compiled 
component loads in browser, cmd prompt disappears
Only way to stop is task manager

**Expected behavior:**
command prompt should stay open showing watch as in previous versions

**Steps to reproduce:**
open windows cmd prompt
create a new component with npm init stencil
change dir to new component
npm i
npm start
component loads in browser, cmd prompt disappears
Only way to stop is task manager

**Related code:**

it is the standard  init project
Hello, World! I'm Stencil 'Don't call me a framework' JS

**Other information:**
windows 10
same behavior with normal cmd prompt or elevated 
npm -v  = 6.14.8

There are no issues when using stencil 1.14

**Comment by splitinfinities** [comment]
Hey, thank you for submitting this! Can you confirm for me if this exists when you use the [Component Starter](https://github.com/ionic-team/stencil-component-starter) repo?

I have to get my windows dev environment set up today. I'll circle back soon.

**Comment by EnterAUserName12** [comment]
I created the project using 

npm init stencil

then chose component

**Comment by splitinfinities** [comment]
Awesome, thank you for the heads up!

**Comment by ianchouinard** [comment]
Someone on our team who uses Powershell & Windows 10 ran into this issue as well. In our case the project was based on the component starter repo and @stencil/core: 2.5.2. 

This issue doesn't happen on other projects we have that are on @stencil/core: 2.0.0

**Comment by rwaskiewicz** [comment]
Hey folks 👋 I've confirmed this is reproducible in v2.5.2 and v2.6.0 when running in Windows. Specifically, I was able to confirm this using both the command prompt & powershell using the Windows 11 Preview (Build 22000.194). This is resolved in v2.7.0 of Stencil, specifically by [this commit](https://github.com/ionic-team/stencil/commit/42cdeaec424fb053648a5ae97e611a7c58d69788). Although the linked commit's message pertains to Docker, updating the underlying dependency appears to fix this issue as well.  Upgrading to the newest version of Stencil (or, v2.7.0 at minimum) should resolve this.  Thanks again!
